packages/api/test/utils/checkAgainstSpec.ts

```diff
@@ -5,31 +5,66 @@ import {applyRecursively, OpenApiJson, parseOpenApiSpec, ParseOpenApiSpecOpts} f
 import {GenericServerTestCases} from "./genericServerTest.js";
 
 const ajv = new Ajv({
-  // strict: true,
-  // strictSchema: true,
   allErrors: true,
 });
 
-// TODO: Still necessary?
+// Ensure embedded schema 'example' do not fail validation
 ajv.addKeyword({
   keyword: "example",
-  validate: () => true,
-  errors: false,
 });
 
 ajv.addFormat("hex", /^0x[a-fA-F0-9]+$/);
 
+/**
+ * An operation that will be filtered during tests execution.
+ */
+export type FilteredOperation = {
+  id: string;
+  /**
+   * When specified, associated operation is tested but `required` properties are ignored from request.
+   */
+  requiredRequestProperties?: string[];
+  /**
+   * When specified, associated operation is tested but `required` query properties are ignored from request.
+   */
+  requiredRequestQueryProperties?: string[];
+  /**
+   * When specified, associated operation is tested but `required` properties are ignored from response.
+   */
+  requiredResponseProperties?: string[];
+};
+
+/**
+ * @returns `true` if the `filteredOperation` is defined and no required property is specified.
+ */
+function isOperationIgnored(filteredOperation: FilteredOperation | undefined): boolean {
```

Comment:
This would require either adding `ignoreOperation` for all operations with no property filtering, which felt a bit verbose to me. 
Also the whole purpose of being in this list is to be filtered, so felt redundant.

Reply:
> This would require either adding ignoreOperation for all operations with no property filtering

Can't this just be optional property? We would use the id to know which route to filter and then look at other values to determine what filtering / ignore rule to apply.

Right now it is not self explanatory if you just look at `filteredOperations` array what is happening, having the jsdoc to explain how it works is good but shouldn't be required to understand the code (if possible)